backend/src/main/java/com/elshop/backend/common/FakerUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.elshop.backend.common;
+
+import com.elshop.backend.product.model.Product;
+import com.github.javafaker.Faker;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class FakerUtils {
+
+    private static final Faker faker = new Faker();
+
+    private static final AppUtils appUtils = new AppUtils();
+
+    private FakerUtils() {
+    }
+
+    public static Product generateFakeProduct(int imagesNr) {
+        List<String> images = generateImagesList(imagesNr);
+        String id = appUtils.generateUuid();
+        String title = faker.commerce().productName();
+        String slug = appUtils.generateSlug(title);
+
+        double price = Double.parseDouble(faker.commerce().price(1, 1000));
+        return new Product(id, slug, title, price, images, "Mobile");
+    }
+
+    public static List<Product> generateFakeProductsList(int productsNr) {
```

Comment:
Der Variablenname `productsNr` erklärt meiner Meinung nach für den Aufrufer noch nicht gut genug, welchen Effekt der Parameter hat.
```suggestion
    public static List<Product> generateFakeProductsList(int numberOfProducts) {
```